_assets/stylesheets/base/_typography.scss

```
@@ -141,6 +141,14 @@
 		font-size: 1.15em;
 		margin: 0 0.25em;
 		padding: 0.25em 0.65em;
+
+    &.highlighter-rouge {

```

@zillou could we change this to:

```suggestion
    &.highlighter-rouge {
      padding: 0.2em 0.4em;
      margin: 0;
      font-size: 80%;
    }
```

---

Doesn't look like that worked as intended @zillou, the result is broken code 😞

---

Fixed with 8ca0d23